modules/react/icon/lib/AppletIcon.tsx

```diff
@@ -13,6 +16,7 @@ export interface AppletIconStyles {
   color?: BrandingColor;
 }
 
+/** @deprecated */
```

Comment:
Like above. Is there an alternative? Anything after `@deprecated` can be Markdown and is considered part of the tag's description until a new tag is encountered.